examples/minimqtt_multipub_simpletest.py

```diff
@@ -0,0 +1,130 @@
+# CIRCUITPY_WIFI_SSID and CIRCUITPY_WIFI_PASSWORD keys
+# with your WiFi credentials. Add your Adafruit IO username and key as well.
+# DO NOT share that file or commit it into Git or other source control.
+ssid = os.getenv("CIRCUITPY_WIFI_SSID")
+appw = os.getenv("CIRCUITPY_WIFI_PASSWORD")
+aio_username = os.getenv("aio_username")
+aio_key = os.getenv("aio_key")
+
+# MQTT Topic
+# Use this format for a standard MQTT broker
+feed_01 = aio_username + "/feeds/BME280-RealTemp"
+feed_02 = aio_username + "/feeds/BME280-Pressure"
+feed_03 = aio_username + "/feeds/BME280-Humidity"
+feed_04 = aio_username + "/feeds/BME280-Altitude"
+
+# Time in seconds between updates (polling)
+# 600 = 10 mins, 900 = 15 mins, 1800 = 30 mins, 3600 = 1 hour
+sleep_time = 900
+
+
+# Converts seconds to minutes/hours/days
+# Attribution: Written by DJDevon3 & refined by Elpekenin
+def time_calc(input_time):
+    if input_time < 60:
+        return f"{input_time:.0f} seconds"
+    if input_time < 3600:
+        return f"{input_time / 60:.0f} minutes"
+    if input_time < 86400:
+        return f"{input_time / 60 / 60:.0f} hours"
+    return f"{input_time / 60 / 60 / 24:.1f} days"
+
+
+# Define callback methods which are called when events occur
+# pylint: disable=unused-argument, redefined-outer-name
+def connect(client, userdata, flags, rc):
+    # Method when mqtt_client connected to the broker.
+    print("| | ✅ Connected to MQTT Broker!")
+
+
+def disconnect(client, userdata, rc):
+    # Method when the mqtt_client disconnects from broker.
+    print("| | ✂️ Disconnected from MQTT Broker")
+
+
+def publish(client, userdata, topic, pid):
+    # Method when the mqtt_client publishes data to a feed.
+    print("| | | Published to {0} with PID {1}".format(topic, pid))
+
+
+# Initialize a new MQTT Client object
+mqtt_client = MQTT.MQTT(
+    broker="io.adafruit.com",
+    port=8883,
+    username=aio_username,
+    password=aio_key,
+    socket_pool=pool,
+    ssl_context=ssl.create_default_context(),
+    is_ssl=True,
+)
+
+# Connect callback handlers to mqtt_client
+mqtt_client.on_connect = connect
+mqtt_client.on_disconnect = disconnect
+mqtt_client.on_publish = publish
+
+while True:
+    # These are fake values, replace with sensor variables.
+    BME280_temperature = 80
+    BME280_pressure = round(1014.89, 1)
+    BME280_humidity = round(49.57, 1)
+    BME280_altitude = round(100.543, 2)
+    print("===============================")
+
+    # Board Uptime
+    print("Board Uptime: ", time_calc(time.monotonic()))
+    print("| Connecting to WiFi...")
+
+    while not wifi.radio.ipv4_address:
+        try:
+            wifi.radio.connect(ssid, appw)
+        except ConnectionError as e:
+            print("Connection Error:", e)
+            print("Retrying in 10 seconds")
+            time.sleep(10)
+    print("| ✅ WiFi!")
+
+    while wifi.radio.ipv4_address:
+        try:
+            # Connect to MQTT Broker
+            mqtt_client.connect()
```

Comment:
This is going to connect (and later) disconnect from the broker repeatedly. This is most likely undesired - single TCP connection is better in terms of resource utilization. The right approach is to connect first and then enter a cycle with `loop()` inside.

Reply:
![Capture](https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/assets/49322231/27c57c81-db4e-45b1-b033-5f1a9aa77afa)

I figured that was the right approach as AdafruitIO /overview seems to like that approach.

Also in this screenshot if the delay isn't provided then only the first publish is recorded.  I'll assume AdafruitIO wants them somewhat spaced apart with delays.

Reply:
Calling loop() is necessary to avoid being disconnected from the server once the keep alive timer expires.

Reply:
This is a bit confusing as that's not how the [simpletest example](https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/main/examples/minimqtt_simpletest.py) works. Connect, publish, disconnect works fine even without increasing the keep alive timer manually in mqtt_client setup.  loop is mainly for checking for messages which I'm not at all interested in if all I want to do is connect & publish. If that's not how it should be done then the simpletest example needs to be corrected to reflect that and the [RTD for MQTT.loop()](https://docs.circuitpython.org/projects/minimqtt/en/stable/api.html#adafruit_minimqtt.adafruit_minimqtt.MQTT.loop) needs to be updated with that bit of information.

I believe it's best practice to disconnect explicitly because I've run into an issue where keeping that connection open while using it with multiple other API's can end up running out of sockets.  Disconnecting closes the socket to be reused by another API.